pkgs/by-name/ma/massdns/package.nix

```
@@ -0,0 +1,33 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, nix-update-script
+}:
+stdenv.mkDerivation rec {
+  pname = "massdns";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "blechschmidt";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-hrnAg5ErPt93RV4zobRGVtcKt4aM2tC52r08T7+vRGc=";
+  };
+
+  makeFlags = [ "PREFIX=$(out)" "PROJECT_FLAGS=-DMASSDNS_REVISION='\"v${version}\"'" ];
+  buildFlags = if stdenv.isLinux then "all" else "nolinux";
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = with lib; {
+    description = "A tool to do DNS resolution on large amounts of domain names";
+    homepage = "https://github.com/blechschmidt/massdns";
+    changelog = "https://github.com/blechschmidt/massdns/releases/tag/v${version}";
+    license = licenses.gpl3Only;
+    maintainers = [ maintainers.geoffreyfrogeye ];
```

This is so that others can easily add themselves to this list in the future.
```suggestion
    maintainers = with maintainers; [ geoffreyfrogeye ];
```